matrix/mat64/shadow.go

```diff
@@ -208,7 +208,7 @@ func rectanglesOverlap(off, aCols, bCols, stride int) bool {
 
 	// Flatten the shifted matrix column positions
 	// so a starts at 0, modulo the common stride.
-	const aFrom = 0
+	// const aFrom = 0
```

Comment:
The part of the comment above "a starts at 0" matches the const. I think the reason I left it out of the logic is that it would have gone in the condition below as `return aFrom <= bTo && bFrom < aTo` (or something like that) which would not have been elided by the compiler.